.github/workflows/python-CI.yml

```diff
@@ -122,8 +118,37 @@ jobs:
           hatch run notebooks:clean
           git diff --exit-code
 
+  build-graphql:
+    name: Build GraphQL Schema
+    needs: changes
+    if: ${{ needs.changes.outputs.phoenix == 'true' }}
+    strategy:
+      matrix:
+        py: [ 3.9, 3.12 ]
+        os: [ ubuntu-latest ]
```

Comment:
What's stopping us from just building on 3.12?

Reply:
Not sure it really matters if 3.9 produces a different schema as long as we have one source of truth.